docs/data-sources/dli_spark_templates.md

```diff
@@ -0,0 +1,153 @@
+---
+subcategory: "Data Lake Insight (DLI)"
+---
+
+# huaweicloud_dli_spark_templates
+
+Use this data source to get the list of the DLI spark templates.
+
+## Example Usage
+
+```hcl
+variable "template_id" {}
+data "huaweicloud_dli_spark_templates" "test" {
+  template_id = var.template_id
+}
```

Comment:
```suggestion
variable "template_id" {}

data "huaweicloud_dli_spark_templates" "test" {
  template_id = var.template_id
}
```
using white line to split the difference blocks

Reply:
done